fixed the errors

mptorz opened the pull request:
I tried running your scripts and I had some errors. After I applied these fixes it works fine for me now :) Great script btw!

mptorz committed:
fixed the errors

gaborvecsei commented:
These fixes are not relevant for others.
There is no need to change `config.ini` and `my_config.ini` as it can be called anything (the user can define it. it can be `my_super_awesome_config.ini`)

For the other changes: Which python did you use? And what is the error message?
It works perfectly fine for Python3

mptorz committed:
minor fix

mptorz commented:
I initially run it with python 2.7. When I run it with Python3 I get this
`Traceback (most recent call last):
  File "wakatime_logger.py", line 80, in <module>
    run_the_program()
  File "wakatime_logger.py", line 68, in run_the_program
    start_date = datetime.strptime(START_DATE, "%Y-%m-%d").date()
TypeError: strptime() argument 1 must be str, not datetime.date
`
This error disappeared after I changed [this line of code](https://github.com/gaborvecsei/Wakatime-Logger/pull/2/commits/f82e4b6ef3943773128286a3239c6b738ce4c6b3#diff-8ee2dbba183b3929ce982dfacee17d2fL14). I also changed "my_config.ini" to "config.ini", because the [file in the folder](https://github.com/gaborvecsei/Wakatime-Logger/blob/master/wakatime-logger-Manual/config.ini) is called "config.ini".

mptorz commented on wakatime-logger-Manual/wakatime_logger.py:
```diff
@@ -38,7 +38,7 @@ def write_data_to_dataframe(df, start_date, end_date):
         start_date = start_date + timedelta(days=1)
     for d in date_range(start_date, end_date):
-        response_json = get_durations_from_waka(d, prepare_request_header(str.encode(API_KEY)))
+        response_json = get_durations_from_waka(d, prepare_request_header(unicode.encode(API_KEY)))
```
This was causing errors to me, because I was using python2. I reversed this change in my next commit
